v2/models/describedobject.go

```diff
@@ -0,0 +1,14 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package models
+
+// DescribedObject is a hold-over from the Java conversion and is supposed to represent inheritance whereby a type
+// with a Description property IS A DescribedObject. However since there is no inheritance in Go, this should be
+// eliminated and the Description property moved to the relevant types. 4 types currently use this.
+type DescribedObject struct {
```

Comment:
Let eliminate this as comment suggests.

Reply:
Removed the DescribedObject, move timestamp and description to the relevant struct.